docs/manuals/userman/gui/viselements/generic/progress.md_template

```diff
@@ -12,4 +12,22 @@ name to select the date selectors on your page and apply style.
 
 # Usage
 
-TODO
+## Circular Progress
```

Comment:
Ok so typically, every element has its first use case addressed as 'Simple'.
We try to capture how the element will be mostly used. The best is to have as few properties as possible.
In your situation, that would be: only *value* is set.

Note that *render* should not be mentioned: it is used in very rare occasions to hide an element from a page. Not the topic of this documentation page.